gcloud/storage/_helpers.py

```diff
@@ -79,10 +79,12 @@ def _patch_property(self, name, value):
     def _set_properties(self, value):
         """Set the properties for the current object.
 
-        :type value: dict
+        :type value: dict or :class:`gcloud.storage.batch._FutureDict`
         :param value: The properties to be set.
         """
         self._properties = value
+        if hasattr(value, 'owner'):
+            value.owner = self
```

Comment:
@tseaver comment from b1b5da0b9b2cb4f5d00e79d10f890abe50337187:

> I don't see the point of the `owner` -- in fact, I don't think there needs to be more than one instance of `_FutureDict` ever created.

-----

Tres, I was hoping that would be possible but struggled to find a way for `Batch.finish` to find the `_PropertyMixin` objects which needed to have `_properties` replaced.

Did you have something in mind?